pysatMadrigal/instruments/methods/madrigal.py

```diff
@@ -58,7 +56,13 @@ def load(fnames, tag=None, sat_id=None, xarray_coords=[]):
         This input is nominally provided by pysat itself. (default='')
     xarray_coords : list
         List of keywords to use as coordinates if xarray output is desired
-        instead of a Pandas DataFrame (default=[])
+        instead of a Pandas DataFrame.  Can build an xarray Dataset with
```

Comment:
```suggestion
        instead of a Pandas DataFrame.  Can build an xarray Dataset with
```

Reply:
Hard for me to see what I changed with this one, I removed the 'with'

Reply:
It didn't register in the suggestion.